content/en/docs/tasks/observability/distributed-tracing/gateway-api/index.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Configure tracing with Telemetry API
+description: How to configure tracing options using Telemetry API.
+weight: 10
```

Comment:
This should not be 10.

We have an issue where there are multiple index.md files with the same weight: 10. At least the overview is listed first (it has 1). Normally we have the items sorted in some manner and they have their own weights. If this page should be listed before the others, then this one could be 8 for example. If it should be last, we could make it 40. 

I leave it up to someone to determine if there should be a better order and update the weight of the individual files to get that desired order by updating the weights in the other peer folders.

Reply:
I update weight to 8,  cause we want end-user use Telemetry API in the future.

Reply:
BTW, current Skywalking/Zipkin has same weight, will fix it in separated PR.

Reply:
Thanks for updating the other weights as well.